### PR TITLE
Upgrade java

### PR DESCRIPTION
**Pull request by GuidoDechamps**
Java version 13, no Kotlin

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 # in a .gitignore file inside the root folder
 # of that language. See Scala/.gitignore for
 # an example. Thanks.
+
+java/.idea/
```
Comment (jbrains) [inline .gitignore:6]: Please move this out of the root `.gitignore` and into `java/.gitignore`.

File: java/pom.xml [hunk 2]
```diff
@@ -8,32 +8,28 @@
...
+                    <target>11</target>
                     <encoding>UTF-8</encoding>
                 </configuration>
             </plugin>
+
         </plugins>
+
     </build>
-</project>
+</project>
```
Comment (jbrains) [inline java/pom.xml:35]: Please don't change the trailing newline at the end of the file.

File: java/pom.xml [hunk 3]
```diff
@@ -8,32 +8,28 @@
...
+                <version>3.8.1</version>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>11</source>
+                    <target>11</target>
                     <encoding>UTF-8</encoding>
                 </configuration>
             </plugin>
+
```
Comment (jbrains) [inline java/pom.xml:31]: Please remove these extra blank lines.

File: java/src/test/java/com/adaptionsoft/games/trivia/SomeTest.java [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
 
 public class SomeTest {
 
-	@Test
-	public void true_is_true() throws Exception {
-		assertTrue(false);
-	}
+    @Test
+    public void true_is_true() throws Exception {
+        fail();
```
Comment (jbrains) [inline java/src/test/java/com/adaptionsoft/games/trivia/SomeTest.java:12]: Please revert this change. There's something delightful, charming, and intentionally annoying about seeing the name `true_is_true()` and then seeing `assertTrue(false)`. :)

**Comment by jbrains** [comment]
If you made these changes, I'd be happy to merge this pull request. Thank you for taking the time to help!

File: java/src/test/java/com/adaptionsoft/games/trivia/SomeTest.java (hunk 4 above)
Comment (GuidoDechamps) [reply java/src/test/java/com/adaptionsoft/games/trivia/SomeTest.java:12]: but... but... the build failed! :-)
